README.md

```diff
@@ -1,15 +1,105 @@
 
 ## Setup
 
-* Open the project and enter your Merchant credentials in `OrderClient` file.
+* Open the project and enter your Merchant credentials in [`OrderClient`](app/src/main/java/com/klarna/sample/payments/api/OrderClient.kt) file.
 * Rebuild the project.
 * Run the `app` configuration.
 
+### Native Integration
+As previously mentioned Klarna Payments is supported in the native integration. The main idea behind Klarna Payments is to give merchants the ability to offer Klarna’s payment methods to customers. Because Klarna doesn’t own the entire checkout flow, a check needs to be performed before an order is created. Our goal with this integration is to offer you the same simple development experience as you have on the web. As such, integrating Klarna Payments consists of four steps:
+
+1. Creating a session.
+2. Rendering the payment view.
+3. Authorizing the session.
+4. Creating an order.
+
+![](https://developers.klarna.com/static/in-app/kp-diagram.png)
+
+###Screenshots
+
+<img src="Screenshots/Screenshot1.png" width="250"/> <img src="Screenshots/Screenshot2.png" width="250"/> <img src="Screenshots/Screenshot3.png" width="250"/>
+
+## Requirements
+* Android 4.4 or later.
+
+## Support
+If you are having any issues running the SDK in the demo app, please create a question on [Stack Overflow](https://stackoverflow.com/questions/tagged/klarna-inapp-sdk) tagged with `klarna-inapp-sdk`. If you think that something is wrong with the demo app itself, please create an issue.
+
+### Developer Documentation
+
+* [Overview](https://developers.klarna.com/documentation/in-app/)
+* [iOS](https://developers.klarna.com/documentation/in-app/ios/)
+* [Android](https://developers.klarna.com/documentation/in-app/android/)
+
+### API Documentation
+*  [iOS](https://htmlpreview.github.io/?https://github.com/klarna/klarna-mobile-sdk/blob/master/docs/ios/index.html)
+*  [Android](https://htmlpreview.github.io/?https://github.com/klarna/klarna-mobile-sdk/blob/master/docs/android/klarna-mobile-sdk/index.html)
+
+## Contributing
+Thank you for reading this and taking time to contribute to Klarna In-App SDK - Android Demo App! Below is a set of guidelines to help you contribute whether you want to report a bug, come with suggestions or modify code.
+
+### How can I contribute?
+#### Reporting Bugs
+This section will guide you through submitting a bug report for Klarna In-App SDK - Android Demo App.
+
+Before submitting a bug report, please check that the issue hasn't been reported before. If you find a *Closed* issue that seem to describe an issue that is similar to what you want to report, open a new issue and link to the original issue in the new one. When you have checked that the issue hasn't been reported before **fill out [the required template](https://github.com/klarna/klarna-mobile-sdk/blob/a55293185f73609df831a26ddff5a66033f21e55/.github/ISSUE_TEMPLATE/bug_report.md)** which will help us resolve the issue faster. 
```

Comment:
Need a template for bug reports in this repo

Reply:
@adrianpalmquist done.